testutil/random.go

```diff
@@ -375,6 +375,35 @@ func RandomAggregateAndProof() *eth2p0.AggregateAndProof {
 	}
 }
 
+func RandomSignedSyncContributionAndProof() altair.SignedContributionAndProof {
+	return altair.SignedContributionAndProof{
+		Message:   RandomSyncContributionAndProof(),
+		Signature: eth2p0.BLSSignature{},
+	}
+}
+
+func RandomCoreSyncContribution() core.SyncContribution {
+	return core.SyncContribution{SyncCommitteeContribution: *RandomSyncCommitteeContribution()}
+}
+
+func RandomSyncContributionAndProof() *altair.ContributionAndProof {
+	return &altair.ContributionAndProof{
+		AggregatorIndex: 0,
```

Comment:
why not a random index? Like

`AggregatorIndex: rand.Uint64()`

Reply:
done